src/SFA.DAS.EmployerIncentives.Commands/ServiceCollectionExtensions.cs

```diff
@@ -183,6 +192,31 @@ public static IServiceCollection AddAccountService(this IServiceCollection servi
             return serviceCollection;
         }
 
+        private static IServiceCollection AddBusinessCentralClient<T>(this IServiceCollection serviceCollection, Func<HttpClient, IServiceProvider, string, int, T> instance) where T : class
+        {
+            serviceCollection.AddTransient(s =>
+            {
+                var settings = s.GetService<IOptions<BusinessCentralApiClient>>().Value;
+
+                var clientBuilder = new HttpClientBuilder()
+                    .WithDefaultHeaders()
+                    .WithApimAuthorisationHeader(settings)
```

Comment:
Just checking this is calling the APIM?  Or should it be using the Managed Identity Authorisation?

Reply:
No it's an APIM call. But not to our APIM (its the BC one)